fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -2017,6 +2040,14 @@ void ReadYourWritesTransaction::setOptionImpl( FDBTransactionOptions::Option opt
 		case FDBTransactionOptions::SPECIAL_KEY_SPACE_RELAXED:
 			validateOptionValue(value, false);
 			options.specialKeySpaceRelaxed = true;
+			break;
+		case FDBTransactionOptions::SPECIAL_KEY_SPACE_CHANGE_CONFIGURATION:
```

Comment:
Maybe we should call this SPECIAL_KEY_SPACE_ENABLE_WRITES?

Reply:
Done. Thanks.